miniapps/navier/navier_solver.cpp

```diff
@@ -290,6 +290,21 @@ void NavierSolver::Setup(double dt)
 
    un_gf.GetTrueDofs(un);
 
+   if (filter_alpha != 0.0)
```

Comment:
Document in `CHANGELOG` and maybe mention the reference in the code?

Reply:
I mentioned the reference. This is the first release with `Navier` officially included so I don't know if this should be mentioned in the `CHANGELOG`.

Reply:
> This is the first release with Navier officially included so I don't know if this should be mentioned in the CHANGELOG.

It is included in 4.2, where it is mentioned... or do you mean something else?

Reply:
Do you want to update the `CHANGELOG` now?